graphql/schema.graphql

```diff
@@ -95,5 +96,17 @@ type Task {
   generatedBy: String
 }
 
+type GroupedProjects {
+  name: String
```

Comment:
This should probably be String!

Reply:
i think i made it nullable because i noticed in the legacy UI, not all of the projects are associated with the group names